### DISCUSSION TITLE
Installing the Jaeger Extension Chart

### DISCUSSION THREAD

**Discussion by kaypee90** [linkerd/linkerd2#7135] [Q&A] [answered]
I am new to linkerd. I am trying to install Jaeger extension with this command `helm install linkerd-jaeger linkerd/linkerd-jaeger` but I get this error:

`Error: unable to build kubernetes objects from release manifest: [unable to recognize "": no matches for kind "Server" in version "policy.linkerd.io/v1beta1", unable to recognize "": no matches for kind "ServerAuthorization" in version "policy.linkerd.io/v1beta1"]`

What am I missing?

**Comment by alpeb** [comment, marked as answer]
`Server` is a resource that was introduced in the latest stable version, `2.11.0`. It looks like you just need to upgrade your linkerd core control plane before attempting to install the latest version of the linkerd-jaeger extension.
